subsys/bluetooth/host/Kconfig

```diff
@@ -335,6 +335,17 @@ config BT_SCAN_WITH_IDENTITY
 	  disclosing local identity information. However, if the use case
 	  requires disclosing it then enable this option.
 
+config BT_DEVICE_NAME_STORAGE
+	int "Size in bytes for device name"
+	default BT_RX_BUF_LEN
+	default BT_L2CAP_RX_MTU if BT_HCI_ACL_FLOW_CONTROL
+	default 0 if !BT_SETTINGS
+	range 0 248
+	help
+	  Bluetooth device name storage size. Storage can be up to 248 bytes
+	  long. 0 means that no storage space is allocated for device name in
+	  which case the device name cannot be changed at runtime.
```

Comment:
You should make it clear here that this includes the terminating null character.

Reply:
Right, I hadn't mentioned it is a string though but I guess one can still assume so it is better to make it clear here.